components/service/glean/src/main/java/mozilla/components/service/glean/EventMetricType.kt

```diff
@@ -62,12 +63,11 @@ data class EventMetricType(
             return
         }
 
-        val truncatedObjectId = objectId.let {
-            if (it.length > MAX_LENGTH_OBJECT_ID) {
-                logger.warn("objectId parameter exceeds maximum string length, truncating.")
-                return@let it.substring(0, MAX_LENGTH_OBJECT_ID)
-            }
-            it
+        // We don't need to check that the objectId is short, since that
```

Comment:
Good point. This means we can get rid of `MAX_LENGTH_OBJECT_ID` as well at the top of this file, right?

Reply:
Good point.